spec/rails_helper.rb

```diff
@@ -35,7 +35,7 @@
   # If you're not using ActiveRecord, or you'd prefer not to run each of your
   # examples within a transaction, remove the following line or assign false
   # instead of true.
-  config.use_transactional_fixtures = true
+  # config.use_transactional_fixtures = true
```

Comment:
Is this actually changing anything? The comment above makes it sound like true is the default...